src/Analyzers/CSharp/Tests/UseCollectionExpression/UseCollectionExpressionForArrayTests.cs

```diff
@@ -4073,4 +4073,99 @@ ReadOnlySpan<int> M(int i)
             ReferenceAssemblies = ReferenceAssemblies.Net.Net80,
         }.RunAsync();
     }
+
+    [Fact]
+    public async Task TestFixAllConditionalExpression1()
+    {
+        await new VerifyCS.Test
+        {
+            TestCode = $$"""
+                class C
+                {
+                    void M(bool b)
+                    {
+                        var v = b ? [|[|new|][]|] { "a" } : [|[|new|][]|] { "b" };
```

Comment:
fix-all was updating both sides of the conditional, leading to broken code (since we do not currently support `x ? [a] : [b]`).  Support for that is coming in c# 13.